### PR TITLE
Add initial POM.xml files

### PR DESCRIPTION
**Pull request by jphalip**

### CODE REVIEW

File: hive-bigquery-parent/pom.xml [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+  <groupId>com.google.cloud.hive</groupId>
+  <artifactId>hive-bigquery-parent</artifactId>
+  <version>${revision}</version>
+  <packaging>pom</packaging>
+  <name>Hive BigQuery Connector Build Parent</name>
+  <description>Parent project for all the Hive BigQuery Connector artifacts
+  </description>
+
+  <properties>
+    <revision>2.0.0</revision>
```
Comment (davidrabinowitz) [inline hive-bigquery-parent/pom.xml:31]: default revision should be 2.0.0-SNAPHOT

File: shaded-dependencies/pom.xml [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+  <!--
+  This module's sole purpose is to shade some libraries (e.g. Guava) to avoid conflicts with other versions that are
+  used by other tools, like Hadoop and Hive. The reason why we shade those libraries in a separate module instead of
+  in the main storage handler module, is to do the shading upstream so that the storage handler's integration tests
+  can run without conflicts.
+  -->
+
+  <dependencies>
+
+    <dependency>
```
[the remark this answers is not in the document]
Comment (davidrabinowitz) [reply shaded-dependencies/pom.xml:39]: should be ok for now, we plan to release 0.25 soon

**Review by davidrabinowitz — CHANGES_REQUESTED** [review]

File: hive-bigquery-parent/pom.xml (hunk 1 above)
Comment (jphalip) [reply hive-bigquery-parent/pom.xml]: Ok, done

**Review by VinayLondhe14 — APPROVED** [review]

**Review by davidrabinowitz — APPROVED** [review]

**Pull request merged**